opm/autodiff/RedistributeDataHandles.hpp

```diff
@@ -387,11 +460,30 @@ void distributeGridAndData( Dune::CpGrid& grid,
                                  geology, distributed_geology);
     grid.scatterData(geo_handle);
 
+    std::vector<double> distributed_pressures;
+
+    if( !threshold_pressures.empty() ) // Might be empty if not specified
+    {
+        if( threshold_pressures.size() !=
+            static_cast<std::size_t>(UgGridHelpers::numFaces(global_grid)) )
+        {
+            OPM_THROW(std::runtime_error, "NNCs not yet supported for parallel runs. "
+                      << UgGridHelpers::numFaces(grid) << " faces but " <<
+                      threshold_pressures.size()<<" pressure values");
```

Comment:
On Thu, Feb 18, 2016 at 09:10:29AM -0800, Atgeirr Flø Rasmussen wrote:
> Minor detail: this error message will be confusing, better to say "threshold pressure values" instead of "pressure values".

Fixed.